app/controllers/usersController.js

```diff
@@ -1,36 +1,27 @@
 const { registerUser } = require('../services/users');
+const { encryptPassword } = require('../helpers/encryption');
+const { isPasswordAlphanumeric, isPasswordLenghtvalid, isEmailValid } = require('../validators/users');
 const logger = require('../logger/index');
 
-// logger.info('hola');
-// logger.error('hola');
-
 const addUser = (req, res) => {
-  const regexAlphanumeric = /^[a-zA-Z0-9]*$/;
-  // eslint-disable-next-line no-useless-escape
-  const regexValidEmail = /^([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)@wolox.com.ar$/;
-
   const { firstName, lastName, email } = req.body;
   let { password } = req.body;
 
-  const bcrypt = require('bcryptjs');
-  const salt = bcrypt.genSaltSync(10);
-  const hash = bcrypt.hashSync(password, salt);
-
-  if (password.length < 8) {
+  if (isPasswordLenghtvalid(password) === false) {
```

Comment:
`if (isPasswordLenghtvalid(password) === false)`  THe `=== false` part is not necesary since your function return a boolean